runtime/lua/vim/fs.lua

```diff
@@ -203,7 +203,11 @@ end
 ---@return (string) Normalized path
 function M.normalize(path)
   vim.validate({ path = { path, 's' } })
-  return (path:gsub('^~/', vim.env.HOME .. '/'):gsub('%$([%w_]+)', vim.env):gsub('\\', '/'))
+  path = (path:gsub('^~/', vim.env.HOME .. '/'):gsub('%$([%w_]+)', vim.env))
+  if vim.fn.has('win32') == 1 then
```

Comment:
using `package.config:sub(1,1)` should be a better idea.
see: https://stackoverflow.com/a/37953903/4438921

Reply:
`vim.fn.has` not works in fast event.  use `vim.fn.windowsversion`  instead this. after #22187 it can be. another choice is  `vim.loop.os_uname().sysname == 'Windows_NT'` . suggest use `windowsversion` . don't use `pacakage.config:sub`.